main.go

```diff
@@ -540,15 +540,15 @@ func (b *Bot) Logf(msg string, args ...interface{}) {
 func main() {
 	var confPath string
 	conf := BotConf{
-		DataPath:      "mattermost.json",
+		DataPath:      "conf/mattermost.json",
 		Channel:       "town-square",
 		DebugChannel:  "test",
 		MaxTweets:     20,
 		CheckInterval: 120,
 	}
 
 	// Parse cmdline flags
-	flag.StringVar(&confPath, "config", "config.json",
+	flag.StringVar(&confPath, "conf/config", "conf/config.json",
```

Comment:
This is a backwards incompatible change.  I don't want people's current installation to break when they do an update.

Reply:
hmm.. yeah.. that's what I didn't like about this. I'm not too well versed in Go. Is there a way that we can keep this backwards compatible but still have the config in a folder?

Docker only allows you to mount folders and the config would need to be customized. Hence my moving it into a folder.
This way we can even publish a pre-built docker image to the docker registry that people can use.

Reply:
I barely know anything about docker, but couldn't you just specify the configuration file path in the `Dockerfile`?

Reply:
that should be possible, yes. 
Give me a few minutes to give this a try